CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## v1.2.0 (2020-09-26)
```

Comment:
why are adding now changelogs?

Reply:
there was no changelog.md for  this integration, I added to unify with the others integrations

Reply:
oh, my bad, thanks for adding it!